docs/design/fabric-v2+/fpc-key-dist.puml

```diff
@@ -12,11 +12,10 @@
 hide footbox
 title Foot Box removed
 
-title Fabric Private Chaincode - Key generation, distribution, confirmation 
+title Fabric Private Chaincode - Chaincode Key generation, distribution, registration 
 
 actor       "Admin_Org1" as Admin1
-participant "ECC_Manager (syscc) Peer_Org1" as ECC_Manager1
-database    "Local storage Peer_Org1" as Storage1
+participant "E_Manager Peer_Org1" as E_Manager1
```

Comment:
I have... renamed Emanager to peer_cli.
At this point, it is not clear (at least to me) what the emanager is, how it could implemented and, therefore, what this document is describing.

Reply:
> At this point, it is not clear (at least to me) what the emanager is, how it could implemented and, therefore, what this document is describing.

I thought the emanager would be the component which deals with persistance of enclave state.  I actually thought we all agreed this morning that this is a component we want to have separate -- that's why i explicitly called out first on an agreement on the set of components before we got into the what-is-the-best-name discussion.  That said, as discussed on slack, i think practically speaking this component will be a part of FPC_stub (storing in the filesystem space provided to external builders), so we could also scrap emanager and use just fpc_stub instead.   That would mean, though, also changing invocation/execution details which otherwise are essentially ready-to-merge ..